### PR TITLE
Suppress warnings

### PR DESCRIPTION
**Pull request by RoboticsYY**
This PR intends to suppress `-Wsign-compare` and `Wunused-variable` warnings showing up [here](https://travis-ci.org/ros-planning/moveit2/builds/606100580#L902) when compiling moveit_core.

### CODE REVIEW

File: test/test_point_inclusion.cpp [hunk 1]
```diff
@@ -241,7 +241,6 @@ TEST(BoxPointContainment, Basic)
   EXPECT_FALSE(box.containsPoint(Eigen::Vector3d(0.00, 1.01, 1.01)));
 
   // near three-axis maximum
-  const double sq3 = sqrt(3) / 3;
```
Comment (henningkayser) [inline test/test_point_inclusion.cpp]: Weird, this seems to belong to the original test (see https://github.com/ros-planning/geometric_shapes/blob/melodic-devel/test/test_point_inclusion.cpp#L95). Probably a hotfix to get this test to pass. Could you just revert this change and fix the original test?

**Review by henningkayser — COMMENTED** [review]
Thanks, looks good! We'll have to fix clang-format after this PR is merged

File: test/test_point_inclusion.cpp (hunk 1 above)
Comment (RoboticsYY) [reply test/test_point_inclusion.cpp]: Yes, these warnings also belong to the original test, which also showed up in the [MoveIt CI](https://travis-ci.org/ros-planning/moveit/jobs/610040749#L1264) result. I filed up a [PR](https://github.com/ros-planning/geometric_shapes/pull/117) to the original test, and temporarily disable these two warnings here until the original test is fixed.

Comment (henningkayser) [reply test/test_point_inclusion.cpp]: Oh, I see.. great catch!

**Comment by RoboticsYY** [comment]
Since the warnings were already fixed in the upstream repository, close this PR.
